flow/connectors/clickhouse/s3_stage.go

```diff
@@ -0,0 +1,83 @@
+package connclickhouse
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/jackc/pgx/v5"
+	"github.com/jackc/pgx/v5/pgxpool"
+
+	utils "github.com/PeerDB-io/peer-flow/connectors/utils/avro"
+	"github.com/PeerDB-io/peer-flow/peerdbenv"
+)
+
+type ClickHouseS3Stage struct{}
```

Comment:
Bit annoying how we can't settle on Clickhouse vs ClickHouse. Should settle on latter in another PR

Reply:
yup, will rename in a latter PR.